omegaconf/grammar_parser.py

```diff
@@ -20,13 +20,13 @@
 # See test `test_match_simple_interpolation_pattern` for examples.
 _id = "[a-zA-Z_]\\w*"  # foo, foo_bar, abc123
 _config_key = f"({_id}|\\$)+"  # foo, $bar, $foo$bar$
-_dot_path_res = f"({_id}(\\.{_id})*)?"  # foo, foo.bar3, foo_.b4r.b0z
-_dot_path_node = f"(\\.)*({_config_key}(\\.{_config_key})*)?"  # foo, .foo.$bar
-_inter_node = f"\\${{\\s*{_dot_path_node}\\s*}}"  # node interpolation
+_node_path = f"(\\.)*({_config_key}(\\.{_config_key})*)?"  # foo, .foo.$bar
+_node_inter = f"\\${{\\s*{_node_path}\\s*}}"  # node interpolation ${foo.bar}
+_resolver_path = f"({_id}(\\.{_id})*)?"  # foo, ns.bar3, ns_1.ns_2.b0z
```

Comment:
```suggestion
_resolver_name = f"({_id}(\\.{_id})*)?"  # foo, ns.bar3, ns_1.ns_2.b0z
```

Reply:
Done in d70a31f (including the corresponding rename down below)